static/css/global.css

```diff
@@ -319,3 +319,45 @@ a:focus {
 .aui-iconfont-warning {
   color: #FFAB00;
 }
+
+/******************************
+Skeleton element styling
+*******************************/
+.skeleton {
+  animation: 1s loading ease-in-out infinite;
+  animation-delay: .06s;
+  background: linear-gradient(
+          100deg,
+          rgba(255, 255, 255, 0) 40%,
+          rgba(255, 255, 255, .5) 50%,
+          rgba(255, 255, 255, 0) 60%
+  ) #ebecf0;
+  background-size: 200% 100%;
+  background-position-x: 180%;
+  border-radius: 4px;
+}
+
+.skeleton,
+.skeleton > label {
+/* Override AUI styling */
+  color: transparent !important;
```

Comment:
I definitely don't like using this but unfortunately the styling of AUI components really get in the way here and override this style without it.